tests/unit/ops/test_ops_schema.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from nvtabular import ColumnSchema, ColumnSelector, Schema, ops
+
+
+@pytest.mark.parametrize("properties", [{}, {"p1": "1"}])
+@pytest.mark.parametrize("tags", [[], ["TAG1", "TAG2"]])
+@pytest.mark.parametrize(
+    "op",
+    [
+        (ops.Bucketize, [1]),
```

Comment:
It seems like this could be simpler if we went ahead and constructed the operator object with these args right here in this list. Then we wouldn't need [the code below that unpacks these tuples](https://github.com/NVIDIA/NVTabular/pull/1079/files#diff-3763c2abd8f8b665145b2bffd223028183a07aa894798c24e37b0e59de3c23fcR33-R39) and passes/splats the args into the operator's constructor.

Reply:
I tried it but it didnt work well... sometimes the op needed to be reinitialized anyway... and this introduced more inconsistencies. So I dont know about this one.